scripts/build.sh

```diff
@@ -100,6 +107,20 @@ KEY_PATH=$SPARKSWAP_DIRECTORY/secure/broker-rpc-tls.key
 CERT_PATH=$SPARKSWAP_DIRECTORY/secure/broker-rpc-tls.cert
 CSR_PATH=$SPARKSWAP_DIRECTORY/secure/broker-rpc-csr.csr
 
+# We want to fail out if the user has both cert flags set in the script
+if [[ "$FORCE_CERTS" == "true" ]] && [[ "$NO_CERTS" == "true " ]]; then
```

Comment:
did you mean to have "$NO_CERTS" == "true " with a space after true? will it make a difference?